### DISCUSSION TITLE
cs.github.com link to provide feedback on itself is stale and doesn't work

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#20411] [Other Feature Feedback, Questions, & Ideas] [answered]
1. Load https://cs.github.com/check-spelling/bats-core
2. See:
    <img width="680" alt="image" src="https://user-images.githubusercontent.com/2119212/178647686-4ecfd57f-9963-427d-b992-17a43107a7ac.png">
    > Doesn't look like check-spelling/bats-core is a known repository. Search [all repos](https://cs.github.com/) instead?
Want to help make code search better? Provide [feedback here!](https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback)
3. Click the link [feedback here!](https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback) link.

## Actual results

https://github.com/github-community/community/discussions/categories/code-search-and-navigation-feedback

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/2119212/178647833-fb55a24b-8f33-4349-990b-4e064f9bb6f0.png">

## Expected results

Links from GitHub software to GitHub forums should be current and work. (The same should apply to those sent by support, but, I digress?)

The correct link is:
https://github.com/github-community/community/discussions/categories/code-search-and-navigation

**Comment by setharnold** [comment]
~~Hmm, perhaps you're the first to give feedback?~~

Sorry, I missed the "the correct link is:" part of your question.

**Comment by look** [comment, marked as answer]
Thanks for the report @jsoref. I've rolled out a fix for this.

**Reply by jsoref** [reply]
Thanks @look, the feedback link works nicely. But shouldn't the code search page not error two days later? How long does it take for CS to learn about a fork?

**Reply by jsoref** [reply]
To be clear, that's a third issue, distinct from this issue (resolved), and from the messaging for #20412
